spec/likeno/entity_spec.rb

```diff
@@ -201,9 +201,12 @@ class EntityTest < Likeno::Entity
   end
 
   describe 'find' do
+    let(:prefix) { '' }
```

Comment:
Why this change? Why not just add these parameter on the next context? These are the default values for these parameters, therefore a perfectly reasonable use for the method is to not explicitly include them on the method call and expect to function exactly as before.